### ISSUE TITLE
Last changes to check if its running in WSL are too strict

### ISSUE THREAD

**Issue by arlllk** [arkane-systems/genie#33]
Using the latest git version, and in the check of 

```
private static bool IsWsl()
        {
            var osrelease = File.ReadAllText("/proc/sys/kernel/osrelease");

            return osrelease.Contains ("Microsoft");
        }
```
isn't case sensitive, I have a recompiled custom kernel and the name that the kernel is 
`4.19.79-microsoft-custom-arllk-g9203c937e-dirty`
from what I see is just to make a caseless comparison

**Comment by cerebrate** [comment]
Yeah, I've been meaning to change that since this morning, but I've had major kernel breakage all day, alas. Will change as soon as I figure out which option breaks .81.

**Comment by arlllk** [comment]
I have seen that, it could be CONFIG_MODULE_FORCE_LOAD,

**Comment by cerebrate** [comment]
Fixed, among other things, in 1.20.
